src/renderer/components/badge/badge.scss

```diff
@@ -6,10 +6,25 @@
   padding: .2em .4em;
   white-space: nowrap;
   max-width: 100%;
-  overflow: hidden;
-  text-overflow: ellipsis;
+  position: relative;
+  cursor: pointer;
+
+  &:hover {
+    background-color: $mainBackground;
+  }
```

Comment:
This is confusing for items that don't have extra text to display.

Reply:
No it's not. Actually it improves contrast/readability of text-label. Just try it in live :)

Reply:
I did, and that is where I noticed it. It is also bad UI to have the mouse change to `"pointer"` when there is no action to be taken.

Reply:
Having `pointer` for some of them which is not actually interactive at all doesn't sound too bad for me.

Reply:
The point of `cursor: pointer` is as an affordance and should be used as one.

Reply:
Ok, removed when the badge considered as "non-interactive"